ros_buildfarm/templates/status/blocked_releases_page.html.em

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>@title - @start_time</title>
+  <meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
+
+  <script type="text/javascript" src="js/moment.min.js"></script>
+  <script type="text/javascript" src="js/zepto.min.js"></script>
+  <script type="text/javascript">
+    window.META_COLUMNS = 8;
+  </script>
+  <script type="text/javascript" src="js/setup.js?@(resource_hashes['setup.js'])"></script>
+  <script type="text/javascript">
+    var QUERY_TRANSFORMS = {
+      'id': 'id=".*"',
+      'label': 'label=".*"',
+    };
+
+    function encoded_query_a_tag(query, title, content) {
+      document.write('<a href="?q=' + encodeURIComponent(query) + '" title="' + title + '">' + content + "</a>")
+    }
+  </script>
+
+  <link rel="stylesheet" type="text/css" href="css/status_page.css?@(resource_hashes['status_page.css'])" />
+  <link rel="stylesheet" type="text/css" href="css/blocked_releases_page.css?@(resource_hashes['blocked_releases_page.css'])" />
+</head>
+<body>
+@{
+import time
+}@
+  <script type="text/javascript">
+    window.body_ready_with_age(moment.duration(moment() - moment("@(time.time())", "X")));
+  </script>
+  <div class="top logo">
+    <h1><img src="http://wiki.ros.org/custom/images/ros_org.png" alt="ROS.org" width="150" height="32" /></h1>
+    <h2>Repos blocked by other repos<br />@(rosdistro_name)</h2>
+  </div>
+  <div class="top search">
+    <form action="?">
+      <input type="text" name="q" id="q" />
+      <p>
+        <a href="?q=" title="Show all repos">all</a>,
+        <script language="JavaScript">encoded_query_a_tag('label="RELEASED"', "Repositories that have already been released", "released")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNRELEASED"', "Repositories that have not been released", "unreleased")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="BLOCKED"', "Repositories that are blocked from being released because of unreleased dependencies", "blocked")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED"', "Repositories that can be released", "releasable")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED_BLOCKING"', "Repositories that can be released and are preventing others from being released", "releasable and blocking")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED_UNBLOCKING"', "Repositories that can be released and are not preventing others from being released", "releasable and not blocking")</script>,
+        <script language="JavaScript">encoded_query_a_tag('id="metapackages"', "Repositories that are dependencies of the metapackages repository", "metapackages")</script>
```

Comment:
it's an english correction, it wasn't for readability (despite what the commit message says - I just lumped the changes together)